CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.2.0] - 2016-05-10
+### Updated
```

Comment:
Can we use one of the changes from http://keepachangelog.com/:

* Added
* Changed
* Deprecated
* Removed
* Fixed
* Security

Perhaps this is `Changed`?